pennylane/ops/qubit/observables.py

```diff
@@ -197,6 +197,18 @@ class SparseHamiltonian(Observable):
         do_queue (bool): Indicates whether the operator should be
             immediately pushed into the Operator queue (optional)
         id (str or None): String representing the operation (optional)
+
+    **Example:**
+        You can construct your Hamiltonian as usual using :class:`~.Hamiltonian` and then utilize
+        the utility function :func:`~.utils.sparse_hamiltonian` to construct the sparse matrix that serves as the input
+        to ``SparseHamiltonian``:
+
+        >>> wires = 20
+        >>> coeffs = [1 for _ in range(wires)]
+        >>> observables = [qml.PauliZ(0) for _ in range(wires)]
```

Comment:
Do you mean
```
[qml.PauliZ(i) for i in range(wires)]
```
Otherwise I believe you're building the Hamiltonian 20 * PauliZ(0), which seems like a weird example

Reply:
That would indeed be a strange example 😄 fixed that, thanks!